pixi.toml

```
@@ -177,7 +177,7 @@ description = "Build SciPy (default settings)"
 
 [feature.build-task.target.win-64.tasks.build]
 cmd = "spin build --setup-args=-Dblas=openblas --setup-args=-Dlapack=openblas --setup-args=-Duse-g77-abi=true && cp .pixi/envs/build/Library/bin/openblas.dll build-install/Lib/site-packages/scipy/linalg/openblas.dll"
-env = { CC = "ccache clang-cl", CXX = "ccache clang-cl", FC = "ccache $FC", FC_LD = "lld-link" }
+env = { CC = "ccache clang-cl", CXX = "ccache clang-cl", FC = "ccache $FC", LDFLAGS = "$CONDA_PREFIX\\lib\\clang\\19\\lib\\windows\\clang_rt.builtins-x86_64.lib" }
```

This can't be the right change. If your goal is to force the C/C++ compilers to also use `lld-link` rather than only the Fortran compiler, then `LD = "lld-link"` should do it.

---

> If your goal is...

This is where I am confused — is the problem, as @ilayn's comment perhaps suggests, that `LDFLAGS` is incompatible with this compiler setup on Windows? Or is it that the wrong linker is picked up?

Note that the unknown argument warnings show *both* on CI and locally for me on main, *despite* the different linker being picked up by meson for C/C++.

---

This whole thing should not be necessary anymore since https://github.com/conda-forge/flang-activation-feedstock/pull/26
```suggestion
env = { CC = "ccache clang-cl", CXX = "ccache clang-cl", FC = "ccache $FC" }
```